cob_default_robot_config/cob4-2/mimic_configurations.yaml

```diff
@@ -0,0 +1 @@
+service_ns: /set_mimic
```

Comment:
Do not use ```service_ns``` to specify service_full_name (see https://github.com/ipa320/cob_command_tools/issues/90)